docs/docs/widgets/datepicker.md

```diff
@@ -89,9 +89,9 @@ Under the <b>General</b> accordion, you can set the value in the string format.
 
 <div style={{paddingTop:'24px'}}>
 
-## Layout
+## Devices
 
-| <div style={{ width:"100px"}}> Layout </div> | <div style={{ width:"100px"}}> Description </div> | <div style={{ width:"135px"}}> Expected Value </div> |
+| <div style={{ width:"100px"}}> Property </div> | <div style={{ width:"100px"}}> Description </div> | <div style={{ width:"135px"}}> Expected Value </div> |
 |:--------------- |:----------------------------------------- | :------------------------------------------------------------------------------------------------------------- |
 | Show on desktop | Toggle on or off to display desktop view. | You can programmatically determining the value by clicking on **fx** to set the value `{{true}}` or `{{false}}`. |
```

Comment:
Hey, here you need to update the content of Description and Expected Value column by referring to the given [reference ](https://docs.tooljet.com/docs/widgets/text-input/#devices)in the issue.

Reply:
Okay let me check it again.